ONSdigital/dp-frontend-router
# Hello World integration

### What

Provides basic integration of the Hello World flow, in preparation for more complete data discovery integration later. Currently routes anything matching `/dd/*` to the Hello World service (dp-dd-frontend-controller).

### How to review

1. Checkout and build via `make debug`
2. Make GET requests to localhost:20000/dd/anything -> should see 502 errors in logs due to the controller service being unavailable.

You can also checkout, build and run the dd-develop branch of dp-dd-frontend-controller and optionally the feature/hello branch of dp-frontend-renderer if you want an end-to-end test. 

### Who can review

Anyone but me.
